controller-server/src/main/java/com/yahoo/vespa/hosted/controller/deployment/JobDetails.java

```diff
@@ -0,0 +1,22 @@
+package com.yahoo.vespa.hosted.controller.deployment;
+
+import com.yahoo.vespa.hosted.controller.api.ActivateResult;
+
+/**
+ * Contains details about a deployment job run.
+ *
+ * @author jonmv
+ */
+public class JobDetails {
+
+    private final ActivateResult deploymentResult;
+    private final String convergenceLog;
```

Comment:
It seems wrong to represent the log as a string. Preferably it should be streamed directly to where it is stored, and if not we need something that allows us to proxy appends to it.

Reply:
This is not meant to be the end result, and suggestions are much appreciated. 

I imagined polling the tester for new output regularly, and stashing that directly. Apparently, our chosen storage does not natively support appends, so it would be a rather stupid get, concat, set. 
Then, to display logs in, e.g., the dashboard, the log would be obtained from the stash, and served again. If it's easy to redirect a stream from storage to client, that'd be great.